customer-stack/customer-stack.yml

```diff
@@ -65,6 +76,11 @@ Conditions:
     - !Equals
       - !Ref kmsKeyArn
       - ""
+  shouldCreateRegionalGateway:
+    !Equals [!Ref apiGatewayType, "REGIONAL"]
+  isVPCConfigPresent: !And
+    - !Equals [!Ref "vpcSubnetIds", ""]
```

Comment:
Since you flipped the condition to "isVPCConfigPresent", you need to also add "NOT" to both equals checks. Right now if vpcSubnetIds == "" and vpcSecurityGroupIds == "" then isVPCConfigPresent = True.

Reply:
Reverted it to the old condition with `Or`.